sql/schema.sql

```diff
@@ -4,12 +4,13 @@ CREATE TYPE roleType AS ENUM ('admin', 'user');
 CREATE TABLE HUser (
 	uid SERIAL PRIMARY KEY,
 	hash text,
-	email text,
+	email text UNIQUE,
```

Comment:
PostgreSQL implicitly creates an index for unique constraints as described [here](http://www.postgresql.org/docs/8.1/static/sql-createtable.html#SQL-CREATETABLE-NOTES). I prefer this over creating an index explicitly.